contract/src/offer-up.contract.js

```diff
@@ -107,15 +107,11 @@ export const start = async zcf => {
       Fail`max ${q(maxItems)} items allowed: ${q(want.Items)}`;
 
     const newItems = itemMint.mintGains(want);
-    atomicRearrange(
-      zcf,
-      harden([
-        // price from buyer to proceeds
-        [buyerSeat, proceeds, { Price: tradePrice }],
-        // new items to buyer
-        [newItems, buyerSeat, want],
-      ]),
-    );
+    const charge = { Price: tradePrice };
+    atomicRearrange(zcf, [
```

Comment:
Please restore the harden

Reply:
Sorry, already stated. (I thought I was commenting on another PR where I hadn't yet said this.)